Set chmod 600 on existing cookieFile

tiagomtotti opened the pull request:
Updates the permission of an existing cookie file to the one expected by Erlang.
This allows the image to work with Kubernetes StatefulSets, when a new POD takes the place of a existing one that was deleted.
Whithout it, an ".erlang.cookie must be accessible by owner only" error will occour, once the Dockerfile sets the entire mount permission to 777.

tianon commented:
So, I don't fully understand what weird thing Kubernetes is doing to cause permissions on this file to reset to 777, but I think this PR could be simpler by just moving the existing `chown` down a line or two outside the `if` it's currently in (and I don't think a comment is necessary).

Also, this needs to be updated in both copies of this script. :+1:

tiagomtotti commented:
@tianon  thanks for your feedback. I've removed the comment, removed the ducplicated line as you suggested  and applied the change in the debian entrypoint as well. 👍

tianon commented:
Looks good, thanks!
